source/Hardpoint.cpp

```diff
@@ -227,10 +230,34 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// to share the same inaccuracy as the projectile.
 	aim += Projectile::Inaccuracy(outfit->Inaccuracy());
 
-	// Create a new projectile, originating from this hardpoint.
-	// In order to get projectiles to start at the right position they are drawn
-	// at an offset of (.5 * velocity). See BatchDrawList.cpp for more details.
-	projectiles.emplace_back(ship, start - .5 * ship.Velocity(), aim, outfit);
+	if(outfit->ShipToShoot().empty())
+		// Create a new projectile, originating from this hardpoint.
+		// In order to get projectiles to start at the right position they are drawn
+		// at an offset of (.5 * velocity). See BatchDrawList.cpp for more details.
+		projectiles.emplace_back(ship, start - .5 * ship.Velocity(), aim, outfit);
+	else
+	{
+		const Ship *shipToPlace = GameData::Ships().Get(outfit->ShipToShoot());
```

Comment:
I feel like all of this could be part of the submunition instead, to spawn the ship when it dies
That way if we want to insta shoot a ship we can just give it 0 lifespan but its not the only option

Reply:
supported